frontend/src/component/layout/MainLayout/NavigationSidebar/NewInUnleash/NewInUnleashItem.tsx

```diff
@@ -21,41 +11,12 @@ export type NewInUnleashItemDetails = {
     onCheckItOut?: () => void;
     docsLink?: string;
     show: boolean;
-    longDescription: ReactNode;
+    longDescription?: ReactNode;
```

Comment:
I've chosen to make this optional. Perhaps the dialog will need some description at some point but it's not clear where that description will go. For now. It's not a thing

Reply:
Maybe this long description was the one hiding the link. Remember?

Reply:
That is actually a bug that I did not solve and still affects other components. Seems to be related to the Beta badge rather than the longDescription but we have no other beta components right now